azure-auth-helper/src/main/java/com/microsoft/azure/auth/Constants.java

```diff
@@ -16,6 +18,14 @@
     public static final String AZURE_HOME_KEY = "AZURE_CONFIG_DIR";
     public static final String AZURE_SECRET_FILE = "azure-secret.json";
 
+    public static final String ERROR = "error";
+    public static final String CODE = "code";
+    public static final String ERROR_DESCRIPTION = "error_description";
+    // TODO: need to update this URL after we post the login quick start at docs.microsoft.com
+    public static final String LOGIN_LANDING_PAGE = "https://docs.microsoft.com/en-us/java/api/overview/azure/maven/azure-webapp-maven-plugin/readme/";
+    public static final int OAUTH_TIMEOUT_MINUTES = 5;
```

Comment:
Why 5?

Reply:
The user spec : If you don't proceed with the login in the browser after 5 minutes, this goal will fail.